lib/elements/footing.rb

```diff
@@ -0,0 +1,29 @@
+module Elements
+  class Footing
+    attr_accessor :step, :length_x, :length_y, :height, :material
```

Comment:
Let's set only getters for this. Usually when a class exposes all its attributes is a red flag, because any other object will be able to know what this one knows.

We can work on how to set each attribute on the go when we need to do it